.github/workflows/ci.yaml

```diff
@@ -55,15 +56,11 @@ jobs:
         python-version: ${{ matrix.python-version }}
         allow-prereleases: true
 
-    - name: Test build
```

Comment:
Is there a reason for removing this step? It's needed to see if the build works on all python versions

Reply:
I will revert. Sorry. Thanks for the heads up.
I was thinking that is enought if we run it for the pypi publish.

Reply:
Starting to review this and noticing this has not yet been reverted - can you do that?

Reply:
Reverted. I forgot about this review.